src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExceptionsPipelineTests.cs

```diff
@@ -421,6 +458,126 @@ public Task EventExceptionsPipeline_ReflectionTypeLoadException()
                 });
         }
 
+        /// <summary>
+        /// Tests the Include filter when a single ExceptionConfiguration is provided.
+        /// </summary>
+        [Fact]
+        public Task EventExceptionsPipeline_IncludeSingle()
+        {
+            return Execute(
+                TestAppScenarios.Exceptions.SubScenarios.SingleException,
+                expectedInstanceCount: 1,
+                validate: instances =>
+                {
+                    IExceptionInstance instance = Assert.Single(instances);
+
+                    ExceptionsConfiguration full = new ExceptionsConfiguration()
+                    {
+                        Include = new() { fullInvalidOperationException }
+                    };
+                    Assert.True(full.ShouldInclude(instance));
```

Comment:
For the Assert.True and Assert.False calls, add a text description as a second parameter describing the failed condition. This way, we can more easily identify from the tests reports what the issue is rather than just seeing "Expeceted: true, Actual: false"

Reply:
I wasn't sure what you wanted for the message, but I figure the only variable thing is what exception we actually capture so that's what I'm showing - you can compare that against the configuration for a particular test to see any potential mismatch. Let me know if you want something different though.